### PR TITLE
add explainer

### PR DESCRIPTION
**Pull request by mkhraisha**
Adds an explainer for TAG #72

### CODE REVIEW

**Comment by mkhraisha** [comment]
not sure why CI is failing on this and #76

File: explainer.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+1. It is possible for verifiers to increase the privacy of the holder whose verifiable credential is being checked by caching status lists that have been fetched from remote servers. By caching the content locally, less correlatable information can be inferred from verifier-based access patterns on the status list.
+2. The use of content distribution networks by issuers can increase the privacy of holders by reducing or eliminating requests for the status lists from the issuer. Often, a request for a revocation list will be served by an edge device and thus be faster and reduce the load on the server as well as cloaking verifiers and holders from issuers.
```
Comment (TallTed) [inline explainer.md:146]:
```suggestion
* It is possible for verifiers to increase the privacy of holders whose verifiable credentials are being checked by caching status lists that have been fetched from remote servers. By caching the content locally, less correlatable information can be inferred from verifier-based access patterns on the status list.
* The use of content distribution networks (CDNs) by issuers can increase the privacy of holders by reducing or eliminating requests for the status lists directly from the issuer. Often, a request for a revocation list can be served by an edge device and thus be faster and reduce the load on the server, as well as cloaking verifiers and holders from issuers.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
The usual language tweaks for clarity

**Comment by mkhraisha** [comment]
Merged in comments

File: explainer.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+This specification describes a privacy-preserving, space-efficient, and high-performance mechanism for publishing status information such as suspension or revocation of Verifiable Credentials.
+
+## Goals
+
+Provide a location for an issuer of a Verifiable Credential [VC-Data-Model](https://www.w3.org/TR/vc-data-model/) to link to a location where a verifier can check to see if a credential has been suspended or revoked.
+
+## How does it work?
+
+At the most basic level, status information for all verifiable credentials issued by an issuer are expressed as simple binary values. The issuer keeps a bitstring list of all verifiable credentials it has issued. Each verifiable credential is associated with a position in the list. If the binary value of the position in the list is 1 (one), the verifiable credential is revoked; if it is 0 (zero), it is not revoked. The verifier can then check the status of a verifiable credential by looking up the position of the verifiable credential in the bitstring list. This is the basic model used for Revocation and Suspension.
```
Comment (msporny) [inline explainer.md:26]: This is no longer accurate as of PR #65.

**Review by msporny — APPROVED** [review]

**Pull request merged**